### PR TITLE
Add CoreOS specific flexvol-driver-host note

### PR DESCRIPTION
**Pull request by syscll**
## Description

As `/usr` on CoreOS is readonly, we should add a note to change the default path of the `flexvol-driver-host` volume.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: _includes/v3.9/coreos-flexvol-driver.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+1. If you are NOT installing on CoreOS, skip to the next step. Otherwise, as `/usr` on CoreOS is readonly, the default
+   path of the `flexvol-driver-host` volume will need to be changed to match the path of the `--flex-volume-plugin-dir`
+   flag passed to the `kube-controller-manager`.
+
+   For example, before:
```
[the remark this answers is not in the document]
Comment (syscll) [reply _includes/v3.9/coreos-flexvol-driver.md]: Totally agree. Removing the before section is probably better!

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 95336957acbfe65a9602d01be553bc01b6b7f0f9

https://deploy-preview-2854--calico.netlify.com

**Comment by syscll** [comment]
If this change is ok I'll port it to all versions.

**Comment by caseydavenport** [comment]
@syscll this looks good to me - we only need to port it to v3.8 and master since it doesn't apply to earlier versions.

**Comment by syscll** [comment]
Added to master and v3.8 docs.

**Comment by lwr20** [comment]
Um @caseydavenport - why wouldn't this apply to v3.9?

**Comment by syscll** [comment]
@lwr20 I think he means as well as v3.9, as my [comment](https://github.com/projectcalico/calico/pull/2854#issuecomment-531970661) implied I was going to port it to every other version too :smile:

File: master/getting-started/kubernetes/installation/calico.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Installing Calico for policy and networking (recommended)
-canonical_url: 'https://docs.projectcalico.org/v3.7/getting-started/kubernetes/installation/calico'
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/kubernetes/installation/calico'
```
Comment (caseydavenport) [inline master/getting-started/kubernetes/installation/calico.md]: I think the canonical URL should be `v3.9` - I don't think it should ever point at master. 

It's meant to point at the latest released version of the doc.

File: v3.8/getting-started/kubernetes/installation/calico.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Installing Calico for policy and networking (recommended)
-canonical_url: 'https://docs.projectcalico.org/v3.7/getting-started/kubernetes/installation/calico'
+canonical_url: 'https://docs.projectcalico.org/v3.8/getting-started/kubernetes/installation/calico'
```
Comment (caseydavenport) [inline v3.8/getting-started/kubernetes/installation/calico.md]: Same here - this should be v3.9.

**Comment by syscll** [comment]
PTAL

**Pull request merged**